topology.json

```diff
@@ -37,9 +37,9 @@
       },
       {
         "tracePercentage": 20,
-        "spansCount": 25,
+        "spansCount": 10,
         "errorRate": 0,
-        "debugRate": 30,
+        "debugRate": 100,
```

Comment:
I know that you set `debugRate=100` to exclude span sampling. After updating `proxy`'s config do you still need it this parameter equals 100?
If no, please, revert this file as well.